crates/fluvio-sc/src/k8/objects/spg_group.rs

```diff
@@ -293,7 +293,7 @@ mod k8_convert {
             volumes.push(VolumeSpec {
                 name: "tls".to_owned(),
                 secret: Some(SecretVolumeSpec {
-                    secret_name: "fluvio-tls".to_owned(), // fixed
+                    secret_name: tls.secret_name.clone().unwrap(),
```

Comment:
we could use unwrap_or_else(|| "tls-secret".to_string()), or change ScOpt to make this a string with a default value of "tls-secret"

Reply:
Ack. Will make that change appropriately.

Reply:
Fixed.